tests/testapp/__init__.py

```diff
@@ -1,3 +1,3 @@
-from .celery import app
+from .celery_app import app
```

Comment:
This change is not related and also not needed. Please revert the change.
```suggestion
from .celery import app
```
You will also need to rename the file again.

Reply:
i couldn't run tests due to circular dependency caused by similarity of this file's name and the celery module. you are right i shouldn't add it to this commit but it slipped out of my hand